hack/testdata/api-rules/violation_exceptions.list

```diff
@@ -5,6 +5,7 @@ API rule violation: names_match,k8s.io/api/core/v1,DaemonEndpoint,Port
 API rule violation: names_match,k8s.io/api/core/v1,Event,ReportingController
 API rule violation: names_match,k8s.io/api/core/v1,FCVolumeSource,WWIDs
 API rule violation: names_match,k8s.io/api/core/v1,GlusterfsVolumeSource,EndpointsName
+API rule violation: names_match,k8s.io/api/core/v1,GlusterfsPersistentVolumeSource,EndpointsName
```

Comment:
I think this file is required to be alphabetical, so move this up one line

Reply:
@roycaihw fyi on usability of the generation error message ^

Reply:
Acknowledge. I'll add sorting before comparing the violations

Reply:
@liggitt Thanks!! yeah, that made the `make update` to work at my end!